src/ext/processObliv/processObliv.ml

```diff
@@ -594,6 +634,12 @@ let rec codegenUncondInstr (instr:instr) : instr = match instr with
         | BNot -> setUnop "__obliv_c__setBitwiseNot" v e loc
         | LNot -> setUnop "__obliv_c__setLogicalNot" v e loc
         end
+    | TFloat(kind, a) when hasOblivAttr a ->
+        begin match op with
+        | Neg -> setUnop "__obliv_c__setNegF" v e loc
+        | BNot -> E.s (E.error "Bitwise operation on float invalid!\n")
```

Comment:
As a style note: the existing errors are not terminated with an exclamation point or a newline (though maybe the newline has a reason?). I would also suggest more specificity, to make it easier for programmers to figure out the problem. Perhaps "bitwise not operator unsupported for obliv-float type". Even better, go find out what error GCC produces and make it read similar to that one if you can (unless their error isn't very good). This comment applies elsewhere in this file as well.